audio/rtptools/Portfile

```diff
@@ -3,10 +3,10 @@
 PortSystem          1.0
 
 name                rtptools
-version             1.21
+version             1.22
 categories          audio net
-license             Noncommercial BSD
-platforms           darwin
+license             BSD
+platforms           darwin openbsd linux solaris
```

Comment:
>Possible values: darwin (= macosx or puredarwin), macosx, puredarwin, freebsd, linux, sunos, netbsd. In general, it can just be set to darwin. (puredarwin is an OS based on Apple's open-source Darwin releases without any of Apple's proprietary bits.) See also os.platform.

Why did you add `openbsd` and `solaris`? They are not on [our guide](https://guide.macports.org/#reference.keywords).

Reply:
tweaked and squashed